From 1f3d3c26a367797915e6f9560631544b278317b9 Mon Sep 17 00:00:00 2001
From: Yelena Krivosheev <yelena@marvell.com>
Date: Thu, 27 Jul 2017 11:20:27 +0300
Subject: [PATCH 1825/2241] arm: defconfig: disable HW buffer manager driver
 for armv7

RX buffers are handled by mvneta driver (software only).

Change-Id: I594c9b897a22744075cdbadb6e47365345c94636
Signed-off-by: Yelena Krivosheev <yelena@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/42281
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
Tested-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm/configs/mvebu_v7_lsp_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/mvebu_v7_lsp_defconfig b/arch/arm/configs/mvebu_v7_lsp_defconfig
index dccc749..7ad6d05 100644
--- a/arch/arm/configs/mvebu_v7_lsp_defconfig
+++ b/arch/arm/configs/mvebu_v7_lsp_defconfig
@@ -84,7 +84,6 @@ CONFIG_E1000E=y
 CONFIG_IXGB=y
 CONFIG_IXGBE=y
 CONFIG_MV643XX_ETH=y
-CONFIG_MVNETA_BM=y
 CONFIG_MVNETA=y
 CONFIG_MVPP2=y
 CONFIG_MVPP3=y
-- 
2.7.4

